src/medications/ViewMedication.tsx

```diff
@@ -226,42 +224,27 @@ const ViewMedication = () => {
         <div className="border-bottom" />
         <Row>
           <Column>
-            <SelectWithLabelFormGroup
-              name="status"
-              label={t('medications.medication.status')}
-              isRequired
+            <Label title="status" text={t('medications.medication.status')} isRequired />
+            <Select
+              id="status"
               options={statusOptionsEdit}
-              defaultSelected={statusOptionsEdit.filter(
-                ({ value }) => value === medicationToView.status,
-              )}
               onChange={(values) => onFieldChange && onFieldChange('status', values[0])}
-              isEditable={isEditable}
```

Comment:
It seems like the `isEditable` field is lost here. Should we pass `!isEditable` as a value to the prop `disabled`?

Reply:
Good call. Probably forgot a couple -- thanks for the reminder!